index.html

```diff
@@ -1027,7 +1031,51 @@ <h1>Adding Videos in HTML</h1>
         <br><code>&lt;video src="img/videohtmlpage.mp4" width="200px" height="120px" type="video/mp4"&gt;example video&lt;/video&gt;</code>
 
         </p>
-        </section>
+    </section>
+
+    <!-- Text Formatting -->
+
+    <section id="text-formatting">
+        <p>
+            <h1>Adding Text Formatting Using HTML</h1>
+            <hr>
+            <br>
+            Text Formatting is done in HTML is done by using these elements.
+            <ul>
+                <li>
+                    <strong>Bold</strong> by using <strong>&lt;strong&gt;&lt;/strong&gt;</strong> or <strong>&lt;b&gt;&lt;/b&gt;</strong> elements.
+                </li>
+                <br>
+                <li>
+                    <em>Italics</em> by using <strong>&lt;em&gt;&lt;/em&gt;</strong> or <strong>&lt;i&gt;&lt;/i&gt;</strong> elements.
+                </li>
+                <br>
+                <li>
+                    <small>Smaller Text</small> by using <strong>&lt;small&gt;&lt;/small&gt;</strong> elements.
+                </li>
+                <br>
+                <li>
+                    <mark>Highlighted Text</mark> by using <strong>&lt;mark&gt;&lt;/mark&gt;</strong> elements.
+                </li>
+                <br>
+                <li>
+                    <strike>Strikethrough</strike> by using <strong>&lt;strike&gt;&lt;/strike&gt;</strong> elements.
```

Comment:
instead of "<strike></strike>" use "<s></s>"
refer to this page https://developer.mozilla.org/en-US/docs/Web/HTML/Element/s